rasa/core/tracker_store.py

```diff
@@ -169,11 +169,14 @@ def __init__(
         password=None,
         event_broker=None,
         record_exp=None,
+        ssl=False,
```

Comment:
optional: I think `use_ssl` is more self explanatory than just `ssl`
```suggestion
        use_ssl: bool =False,
```